pvlib/tests/test_irradiance.py

```diff
@@ -1235,6 +1257,12 @@ def test_clearsky_index():
     assert_series_equal(out, expected)
 
 
+def test_clearsky_index_clearsky_ghi_deprecation():
```

Comment:
```suggestion
@fail_on_pvlib_version("0.12")
def test_clearsky_index_clearsky_ghi_deprecation():
```
At some point during the decorator PR (#2237) I proposed adding this to the test, so removing the decorator in the future is not forgotten. But it didn't draw any attention. What do maintainers think about it? @kandersolar

Sry Dax for the review in dribs and drabs.

Reply:
> Sry Dax for the review in dribs and drabs.

it's totally fine! I'll commit this change for now then if others disagree we could always revert the change

Reply:
> Question though, v0.12? is it not 12.0?

Yes, see the other uses of `@fail_on_pvlib_version`. It is due to its implementation. Intuitively the leftmost part is more important than the last numbers. `major.minor.patch` is the usual way to state them. You wouldn't want something to fail both on pvlib `0.12.0` and `1.12.0`.